NuGet.config

```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <solution>
+    <add key="disableSourceControlIntegration" value="True" />
```

Comment:
I don't understand this line. According to [the reference](https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#solution-section), it indicates

> whether to ignore the packages folder when working with source control.

I don't understand this. NuGet has nothing to do with VC (Git in our case). We use a `.gitignore` for that. I have probably misunderstood this setting, but I don't understand why it would be needed.